src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,131 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75F;
+    private int size = 0;
+    private MyNode<K,V>[] bucketArray = new MyNode[DEFAULT_INITIAL_CAPACITY];
+
+    class MyNode<K,V> {
+        private int hash;
```

Comment:
Do you use this variable?

Reply:
I don't use it. It is not necessary. But i can add some IFs.

Reply:
you can remove this variable)